app/views/search/_granule_details.html.erb

```diff
@@ -3,7 +3,7 @@
 <section id="granule-details" class="details master-overlay-no-min">
   <nav class="master-overlay-nav">
     <a href="#" class="master-overlay-back" data-bind="click: ui.collectionsList.focused().hideGranuleDetails">
-      <i class="fa fa-arrow-circle-left"></i> <span class="master-overlay-breadcrumb">Back to Granules</span>
+      <i class="fa fa-chevron-circle-left"></i> <span class="master-overlay-breadcrumb">Back to Granules</span>
```

Comment:
Missed removing the "report a metadata problem" link. Look at lines 19-26 of this file.